data/brands/shop/supermarket.json

```diff
@@ -194,41 +194,61 @@
         "shop": "supermarket"
       }
     },
-    {
-      "displayName": "Aldi",
-      "id": "aldi-883bfe",
-      "locationSet": {
-        "include": ["150", "au", "us"]
-      },
-      "tags": {
-        "brand": "Aldi",
-        "brand:wikidata": "Q125054",
-        "brand:wikipedia": "en:Aldi",
-        "name": "Aldi",
-        "shop": "supermarket"
-      }
-    },
     {
       "displayName": "Aldi Nord",
       "id": "aldinord-0a839e",
-      "locationSet": {"include": ["de"]},
+      "locationSet": {"include": ["de","fr","es","pt","pl","dk","be","nl","lu"]},
       "tags": {
         "brand": "Aldi Nord",
+		"brand:fr": "Aldi",
+		"brand:es": "Aldi",
+		"brand:pt": "Aldi",
+		"brand:pl": "Aldi",
+		"brand:dk": "Aldi",
+		"brand:be": "Aldi",
```

Comment:
Do you really need to add so many language versions? Especially because most of them are the same? Just keep the ```brand```？
（only my advice）

Reply:
There might be other options, but I thought this would be the most easy one:

- 1 record for ALDI Süd
- 1 record for ALDI Nord 

Another option is using

- 1 record for ALDI Süd in Germany (brand="ALDI Süd")
- 1 record for ALDI Nord in Germany (brand="ALDI Nord)
- 1 record for ALDI Süd for all countries except Germany (brand="ALDI")
- 1 record for ALDI Nord for all countries except for Germany (brand="ALDI")

But then it's 4 records instead of 2 records.

Since it's really two completly independent companies, we need a solution as the current tagging scheme is error prone in Germany where both companies are active.